.github/workflows/ca-ds-connection-test.yml

```diff
@@ -109,6 +109,7 @@ jobs:
           # enrollment should fail
           docker exec pki pki \
               -n caadmin \
+              --ignore-cert-status OCSP_SERVER_ERROR \
```

Comment:
I'm wondering if we should set up a separate OCSP server with it's own DS so that it's not affected by CA DS shutdown which what we're primarily trying to test here.

Reply:
I guess this could be a scenario also for IPA so it is OK to test with internal OCSP